### PR TITLE
5.1.4 Release Notes

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: release-notes/release-notes514.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+> This is the {{site.cloudname}} and {{site.enterpriseversion}} release notes. For information on the latest community version of {{site.productname}}, see: [{{site.productname}} Changelog]({{site.baseurl}}/changelog/).
+
+## General bug fixes
+
+{{site.productname}} 5.1.4 provides fixes for the following bugs:
+
+- The dialog contents disappearing when clicking a checkbox for right-to-left languages.
+- The `legacyoutput` plugin registering legacy formats after editor initialization, causing legacy content to be stripped on the initial load.
+- The search and replace not cycling through results when searching using special characters.
```
Comment (lnewson) [inline release-notes/release-notes514.md]: "Search and replace plugin"?

Comment (tylerkelly13) [reply release-notes/release-notes514.md]: fixed

File: release-notes/release-notes514.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: TinyMCE 5.1.4
+title_nav: TinyMCE 5.1.4
+description: Release notes for TinyMCE 5.1.4
+keywords: releasenotes bugfixes
+---
+
+These release notes provide a high-level coverage of the changes for {{site.productname}} 5.1.4, including:
```
Comment (metricjs) [inline release-notes/release-notes514.md]: "a high-level coverage"?

Comment (tylerkelly13) [reply release-notes/release-notes514.md]: Yeah, that's horrible. updated s/ a high-level coverage/an overview/

**Pull request merged**